src/main/java/com/axibase/tsd/client/SelfSignedTrustManager.java

```diff
@@ -0,0 +1,62 @@
+package com.axibase.tsd.client;
+
+import lombok.extern.slf4j.Slf4j;
+
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+import javax.net.ssl.X509TrustManager;
+import java.io.IOException;
+import java.io.Serializable;
+import java.net.Socket;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+@Slf4j
+public class SelfSignedTrustManager implements X509TrustManager, Serializable, Cloneable {
+    private final static  String CONSTANT = "message";
+
+    public SelfSignedTrustManager() throws NoSuchAlgorithmException, KeyStoreException {
+    }
+
+    @Override
+    public X509Certificate[] getAcceptedIssuers() {
+        return new X509Certificate[0];
+    }
+
+    public void checkServerTrusted(X509Certificate[] chain,
+                                   String authType) throws CertificateException {
+        Socket socket = new Socket();
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use try-with-resources or close this "Socket" in a "finally" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AS2095)